### DISCUSSION TITLE
Running load-test with bruno

### DISCUSSION THREAD

**Discussion by ahoehma** [usebruno/bruno#1218] [Ideas]
It would be cool to be able to 

- define a "load-test-setup" : how many parallel "users", ramp-up-time, waiting-time etc.
- run single request with a "load-setup"
- run collections with a "load-setup"
- got some nice reports at the end

**Comment by gintsgints** [comment]
Do we really need this functionality in UI? May be cli only?
That would be like specify --users option and get provided tests executed.
Example here - https://github.com/usebruno/bruno/pull/4786

**Comment by jhoover4** [comment]
I'd be cool to have load testing functionality, but I'm not sure how much work would be needed to get it setup. @gintsgints what would be needed after the parallelization?

**Reply by gintsgints** [reply]
I have made DRAFT mr how it would look - https://github.com/usebruno/bruno/pull/4786 . 
Example run:

```
node ../../packages/bruno-cli/bin/bru.js run <test_folder> --env <env> --users 10 --ramp-up-time 1000
```

**Reply by gintsgints** [reply]
What is most difficult here is adjusting reporters to summarize different runs results. This is not done here.
